### PR TITLE
Add agenda/schedule

### PR DESCRIPTION
**Pull request by fmvilas**
Preview deploy: https://deploy-preview-7--competent-leakey-e1cd69.netlify.com/

### CODE REVIEW

File: site/index.html [hunk 1]
```diff
@@ -327,7 +327,211 @@ <h5>When</h5>
...
+                                    </div>
+                  <!--Bootstrap card-->
+                  <article class="card card-custom card-corporate">
+                    <div class="card-header" role="tab">
+                      <div class="card-title"><a class="collapsed" id="accordion1-card-head-hfhsoskt" data-toggle="collapse" data-parent="#accordion1" href="#accordion1-card-body-sartmhlo" aria-controls="accordion1-card-body-sartmhlo" aria-expanded="false" role="button"><span class="schedule-classic"><span class="unit unit-spacing-md align-items-center d-block d-md-flex"><span class="unit-left"><span class="schedule-classic-img"><img src="images/speakers/paul-taylor.png" alt=""/></span></span><span class="unit-body"><span class="schedule-classic-content"><span class="schedule-classic-time">11:00 am to 11:40 am (UTC)</span><span class="schedule-classic-title heading-4">Unhappy Path & Dealing with Bad Events</span><span class="schedule-classic-author">by <span class="schedule-classic-author-name">Paul Taylor</span> -  Strategic Advisor at Mulesoft</span></span></span></span></span></a></div>
```
Comment (derberg) [inline site/index.html]:
```suggestion
                      <div class="card-title"><a class="collapsed" id="accordion1-card-head-hfhsoskt" data-toggle="collapse" data-parent="#accordion1" href="#accordion1-card-body-sartmhlo" aria-controls="accordion1-card-body-sartmhlo" aria-expanded="false" role="button"><span class="schedule-classic"><span class="unit unit-spacing-md align-items-center d-block d-md-flex"><span class="unit-left"><span class="schedule-classic-img"><img src="images/speakers/paul-taylor.png" alt=""/></span></span><span class="unit-body"><span class="schedule-classic-content"><span class="schedule-classic-time">11:10 am to 11:40 am (UTC)</span><span class="schedule-classic-title heading-4">Unhappy Path & Dealing with Bad Events</span><span class="schedule-classic-author">by <span class="schedule-classic-author-name">Paul Taylor</span> -  Strategic Advisor at Mulesoft</span></span></span></span></span></a></div>
```

Comment (derberg) [reply site/index.html]: I cut initial proposal time to 30 like all the others, so this one should start at 11.10

**Comment by derberg** [comment]
I also think if maybe providing such a link for each talk would make sense https://everytimezone.com/s/53f38ba3

This is pretty unusual event cross time zones, people might confuse hours

**Comment by fmvilas** [comment]
You mean on each talk title?

**Comment by derberg** [comment]
@fmvilas yes :( I know it is a lot of work, but I can help. Only if we agree it makes sense though, but looking on our proposals list how different people provided info about timezones differently, I think it is needed. Not so urgent to block this PR though

**Pull request merged**